src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/record-jserror-exception.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: recordJSErrorException (Android SDK API)
+type: apiDoc
+shortDescription: Record JavaScript Error for ReactNative apps
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Android SDK API
+metaDescription: 'New Relic API for Android mobile app monitoring: record jserror for react native apps.'
+redirects:
```

Comment:
@ywang-nr this appears to be a new doc, so you shouldn't need any redirects because nobody has been looking for this elsewhere.

Reply:
got it, thank you